app/src/locale.cc

```diff
@@ -59,13 +59,11 @@ std::string GetLocale() {
   // of the LANG or LC_CTYPE environment variables if set, or otherwise return a
   // default locale (empty in real life, or placeholder when running in a unit
   // test, as the test environment has no locale variables set).
-  std::string output = std::locale().name() != "C"
-                           ? std::locale().name()
-                           : getenv("LANG")
-                                 ? getenv("LANG")
-                                 : getenv("LC_CTYPE")
-                                       ? getenv("LC_CTYPE")
-                                       : getenv("TEST_TMPDIR") ? "en_US" : "";
+  std::string output = std::locale().name() != "C" ? std::locale().name()
```

Comment:
nit: This is a case where I think the formatter is going to do worse than the nicely formatted table no matter what settings we use. If we go through with committing with these settings, can this block get wrapped in `// clang-format off/on`?

Reply:
Done.